frontend/assets/koblenz/l10n/override_de.json

```diff
@@ -0,0 +1,47 @@
+{
+  "identification": {
+    "activateCurrentDeviceDescription": "Ihr Pass wurde bereits auf einem anderen Gerät aktiviert. Wenn Sie fortfahren, wird Ihr Pass automatisch auf dem anderen Gerät deaktiviert, falls er dort noch aktiv ist.",
+    "activateCurrentDeviceTitle": "Pass auf diesem Gerät aktivieren?",
+    "activateDescription": "Sie haben den Koblenz-Pass bereits beantragt und einen Aktivierungscode erhalten? Scannen Sie den Code hier ein.",
+    "activateTitle": "Pass aktivieren",
+    "applyDescription": "Sie haben noch keinen Koblenz-Pass? Hier können Sie Ihren Koblenz-Pass beantragen.",
```

Comment:
as I understood, it should be KoblenzPass everywhere

Reply:
Yes